src/pages/Main/Footer/FootSlide.js

```diff
@@ -0,0 +1,99 @@
+import React, { Component } from 'react';
+import { BiRadioCircle, BiRadioCircleMarked } from 'react-icons/bi';
+import './FootSlide.scss';
+
+export class FootSlide extends Component {
+  constructor() {
+    super();
+    this.setSlideRef = React.createRef();
+    this.state = {
+      slideCardNumber: 0,
+      slideList: [
+        {
+          id: 0,
+          icon: 'http://localhost:3000/images/supplements.png',
+          alt: 'supplements',
+          description: 'villy 프로젝트 화이팅!!',
+          user: '도화동 사시는 김태영님',
+        },
+        {
+          id: 1,
+          icon: 'http://localhost:3000/images/vitamin.png',
+          alt: 'vitamin',
+          description: '벡엔드 화이팅!!',
+          user: '공덕동 사시는 설지우님',
+        },
+        {
+          id: 2,
+          icon: 'http://localhost:3000/images/vitamins.png',
+          alt: 'vitamins',
+          description: '프론트엔드 화이팅!!!',
+          user: '안산 사시는 최명준님',
+        },
+      ],
+    };
+  }
+
+  componentDidMount() {
+    this.setSlideRef.current = setInterval(this.autoSlide, 2500);
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.setSlideRef.current);
+  }
+
+  autoSlide = () => {
+    const { slideCardNumber, slideList } = this.state;
+    const nextSlideCardNumber =
+      slideCardNumber > slideList.length - 2 ? 0 : slideCardNumber + 1;
+    this.setState({ slideCardNumber: nextSlideCardNumber });
+  };
+
+  slideHandler = e => {
+    const value = e.currentTarget.value;
+    this.setState({ slideCardNumber: Number(value) });
```

Comment:
한번만 쓰니까 굳이 value라는 변수 선언 안해줘도 될 것 같네요
```suggestion
    this.setState({ slideCardNumber: Number(e.currentTarget.value) });
```

Reply:
넵!